pyproject.toml

```diff
@@ -71,7 +71,10 @@ Documentation = "https://stfc.github.io/janus-core/"
 [dependency-groups]
 dev = [
     "coverage[toml]<8.0.0,>=7.4.1",
+    "data-tutorials>=0.3.1",
     "ipykernel>=6.29.5",
+    "jupyter>=1.1.1",
+    "nbsphinx>=0.9.6",
```

Comment:
Does this want to go in dev or docs?

Reply:
Well spotted, yes I think they ought to be in docs, I'll move them.